PDF_UA/2/8.9 Annotations/8.9.2 Semantics and content/8.9.2.2 Annotations as artifacts/verapdf-profile-8-9-2-2-t01.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<profile xmlns="http://www.verapdf.org/ValidationProfile" flavour="PDFUA_2">
+    <details creator="veraPDF Consortium" created="2023-03-13T10:53:06.894+03:00">
+        <name>ISO 14289-2:202x - 8.9 Annotations - 8.9.2.2 Annotations as artifacts - Invisible flag</name>
+        <description>Annotations shall be artifacts, if the Invisible flag is set in accordance with ISO 32000-2:2020, 
+            Table 167</description>
+    </details>
+    <hash></hash>
+    <rules>
+        <rule object="PDAnnot">
+            <id specification="ISO_14289_2" clause="8.9.2.2" testNumber="1"/>
+            <description>Annotations shall be artifacts, if the Invisible flag is set in accordance with ISO 32000-2:2020,
+                Table 167</description>
+            <test>structParentType == null || isArtifact == true || (F &amp; 1) == 0</test>
+            <error>
+                <message>An annotation is included in logical structure with tag %1 (standard type = %2, Invisible = %3)</message>
```

Comment:
An invisible annotation is included in logical structure with tag %1 (standard type = %2, Invisible = %3)